source/FPSciServerApp.cpp

```diff
@@ -82,6 +82,7 @@ void FPSciServerApp::initExperiment() {
 
 void FPSciServerApp::onNetwork() {
     /* None of this is from the upsteam project */
+    const shared_ptr<PlayerEntity>& player = scene()->typedEntity<PlayerEntity>("player");
```

Comment:
When you start to need to get a typed entity every single loop of `oneFrame()` on the server it might be easier to hang onto a reference to `player` as a member field. However, I'd question if you want to do this using the `PlayerEntity` vs the `PlayerConfig`

Reply:
Removed this. I'll now be reading from the PlayerConfig instead of PlayerEntity.